mappings/net/minecraft/data/server/recipe/TransformSmithingRecipeJsonBuilder.mapping

```diff
@@ -0,0 +1,33 @@
+CLASS net/minecraft/class_8074 net/minecraft/data/server/recipe/TransformSmithingRecipeJsonBuilder
+	FIELD field_42090 template Lnet/minecraft/class_1856;
+	FIELD field_42091 base Lnet/minecraft/class_1856;
+	FIELD field_42092 addition Lnet/minecraft/class_1856;
+	FIELD field_42093 category Lnet/minecraft/class_7800;
+	FIELD field_42094 result Lnet/minecraft/class_1792;
+	FIELD field_42095 advancement Lnet/minecraft/class_161$class_162;
+	FIELD field_42096 serializer Lnet/minecraft/class_1865;
+	METHOD <init> (Lnet/minecraft/class_1865;Lnet/minecraft/class_1856;Lnet/minecraft/class_1856;Lnet/minecraft/class_1856;Lnet/minecraft/class_7800;Lnet/minecraft/class_1792;)V
+		ARG 1 serializer
+		ARG 2 template
+		ARG 3 base
+		ARG 4 addition
+		ARG 5 category
+		ARG 6 result
+	METHOD method_48534 validate (Lnet/minecraft/class_2960;)V
+		ARG 1 id
+	METHOD method_48535 create (Lnet/minecraft/class_1856;Lnet/minecraft/class_1856;Lnet/minecraft/class_1856;Lnet/minecraft/class_7800;Lnet/minecraft/class_1792;)Lnet/minecraft/class_8074;
+		ARG 0 template
+		ARG 1 base
+		ARG 2 addition
+		ARG 3 category
+		ARG 4 result
+	METHOD method_48536 criterion (Ljava/lang/String;Lnet/minecraft/class_184;)Lnet/minecraft/class_8074;
+		ARG 1 name
+		ARG 2 conditions
+	METHOD method_48537 offerTo (Ljava/util/function/Consumer;Lnet/minecraft/class_2960;)V
+		ARG 1 exporter
+		ARG 2 id
+	METHOD method_48538 offerTo (Ljava/util/function/Consumer;Ljava/lang/String;)V
+		ARG 1 exporter
+		ARG 2 id
+	CLASS class_8075 TransfromSmithingRecipeJsonProvider
```

Comment:
Misspelling, plus the id is `smithing_transform`, would be best to keep that naming order